addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -580,12 +595,6 @@ function removePresetGradient(element) {
     }
 }
 
-function setBackgroundImageAndOverride(el, backgroundImage) {
-    const isNone = !backgroundImage || backgroundImage === "none";
-    el.style.backgroundImage = isNone ? "" : backgroundImage;
-    // If the current background image is empty but the inherited one isn't
-    // force the background image to override the inherited one.
-    if (isNone && getComputedStyle(el).backgroundImage !== "none") {
-        el.style.backgroundImage = "none";
-    }
+function setBackgroundImage(el, backgroundImage) {
+    el.style.backgroundImage = backgroundImage;
```

Comment:
Now that I see this, I guess there is no point in having it a function anymore so you can remove it and assign `backgroundImage` directly at call site.